schema/schema.sql

```diff
@@ -64,7 +64,7 @@ CREATE TABLE gutpcomment (
   content varchar not null,
   author_id varchar not null,
   post_id varchar not null,         -- the post this comment belongs to
-  rcomment_id varchar not null,     -- the replied comment id, if has
+  reply_id varchar not null,     -- the replied comment id, if has
```

Comment:
`rcomment_id` indicates the comment replied by another comment. `reply_id` is not so obvious.

Reply:
how about `parent_comment_id`?

Reply:
updated